### PR TITLE
found the origin of the insertion of massive data, got rid of it once…

### PR DESCRIPTION
**Pull request by risingBirdSong**
… and for all !!

@uxsasukexnaruto Hey when you have a chance can you review this please? Thank you!

### CODE REVIEW

File: data_generator/arangoPushTest.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const fs = require("fs");
+const Database = require("arangojs").Database;
+const aql = require("arangojs");
+const db = new Database("http://127.0.0.1:8529");
+db.useBasicAuth("root", "datasaver");
```
Comment (hanguyen0) [inline data_generator/arangoPushTest.js]: make sure to put your username and pass to an env file and gitignore it

File: data_generator/arangoPushTest.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+//         c: Date(),
+//       };
+//       await collection.save(doc);
+//     }
+//     console.log(
+//       "all ---------------------",
+//       collection.all().then((time) => console.log("hey----", time))
+//     );
+//   });
+// }
```
Comment (hanguyen0) [inline data_generator/arangoPushTest.js:41]: probably take out the comments if  you don't use them anymore

File: data_generator/streamWriterLocation.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+const fileName = "./actualData-Locations-workingEdition.json";
+const faker = require("faker");
+
+const { Client } = require("pg");
+const client = new Client({
+  host: "localhost",
+  port: 5432,
+  database: "airbnb",
+  user: "postgres",
+  password: "datasaver",
```
Comment (hanguyen0) [inline data_generator/streamWriterLocation.js:13]: here too

**Review by hanguyen0 — APPROVED** [review]
done

File: data_generator/arangoPushTest.js (hunk 1 above)
Comment (risingBirdSong) [reply data_generator/arangoPushTest.js]: Thanks, good idea

**Pull request merged**